javascript/src/test/components/ChannelMessages/MessageList.test.js

```diff
@@ -1,91 +1,24 @@
 import React from "react";
 import { render } from "@testing-library/react";
-import MessageListView from "main/components/ChannelMessages/MessageListView";
-import useSWR from "swr";
-import MessageScrollableView from "../../../main/components/ChannelMessages/MessageScrollableView";
-jest.mock("swr");
-jest.mock("react-router-dom", () => {
-    return {
-        'useParams': jest.fn(),
-    };
-});
-describe("MessageListView tests", () => {
```

Comment:
It looks like you're deleting a bunch of tests that were put in by other teams. We don't want to undo the work that the other teams did, so can you restore the tests in this file?

I can see that the tests originally in this file were for the `MessageListView` component and that this test file is named `MessageList.test.js`, which breaks our naming convention. This was probably a mistake on their part, but we probably shouldn't be removing their tests. It would be nice if you could reinstate them in a file `MessageListView.test.js`.